hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/scale/AbstractSTestS3AHugeFiles.java

```diff
@@ -456,7 +456,7 @@ public void test_040_PositionedReadHugeFile() throws Throwable {
   }
 
   @Test
-  public void test_045_VectoredIOHugeFile() throws Throwable {
+  public void test_045_VectoredIoHugeFile() throws Throwable {
```

Comment:
capital IO is OK here; your choice

Reply:
Yeah I thought that was the problem but don't know why checkstyle is crying 
`./hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/scale/AbstractSTestS3AHugeFiles.java:459:  public void test_045_vectoredIoHugeFile() throws Throwable {:15: Name 'test_045_vectoredIoHugeFile' must match pattern '^[a-z][a-zA-Z0-9]*$'. [MethodName]`